content-repo/gendocs.py

```diff
@@ -932,9 +932,9 @@ def main():
     if os.getenv('UPDATE_PACK_DOCS') or os.getenv('CI'):
         # to avoid cases that in local dev someone might checkin the modifed pack-docs.md we do this only if explicityl asked for or in CI env
         insert_approved_tags_and_usecases()
-
-    print("Writing json links into contentItemsDocsLinks.json")
-    update_contributors_file(create_service_account_file().name, json.dumps(DOCS_LINKS_JSON, indent=4))
+    if os.getenv('UPDATE_TOP_CONTRIBS') or os.getenv('CI'):
```

Comment:
@Shellyber FYI. Added this so it is possible to run the build locally.

Reply:
Hey @glicht 
This is not connected to the `contribution` file.
This is the `contentItemsDocsLinks` file
Since we moved the marketplace to a separate repo and separate build we upload the `contentItemsDocsLinks` to the bucket during the docs build and then the marketplace build takes it from there.
I really have to change the function name....